packages/compiler/test/server/completion.test.ts

```diff
@@ -71,6 +72,105 @@ describe("compiler: server: completion", () => {
     );
   });
 
+  it("complete import for relative path", async () => {
+    const completions = await complete(` import "./┆ `, undefined, {
+      "test/bar.cadl": "",
+      "test/foo.cadl": "",
+      "test/foo/test.cadl": "",
+    });
+    check(
+      completions,
+      [
+        {
+          label: "bar.cadl",
+          commitCharacters: [],
+          kind: CompletionItemKind.File,
+        },
+        {
+          label: "foo.cadl",
+          commitCharacters: [],
+          kind: CompletionItemKind.File,
+        },
+        {
+          label: "foo",
+          commitCharacters: [],
+          kind: CompletionItemKind.Folder,
+        },
+      ],
+      {
+        allowAdditionalCompletions: false,
+      }
+    );
+  });
+
+  it("complete import for relative path excludes node_modules", async () => {
+    const completions = await complete(` import "./┆ `, undefined, {
+      "test/node_modules/test.cadl": "",
+      "test/main/node_modules/test.cadl": "",
```

Comment:
The node_modules here isn't significant to the test, maybe test/main/test.cadl would be clearer.